_config.yml

```diff
@@ -105,7 +86,54 @@ source_releases:
 
 stable_releases:
   -
-      name: "Apache Flink 1.7.2 only"
+      name: "Apache Flink 1.8.0"
+      version_short: 1.8
+      scala_211:
+          id: "180-download_211"
+          url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.11.tgz"
+          asc_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.11.tgz.asc"
+          sha512_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.11.tgz.sha512"
+      scala_212:
+          id: "180-download_212"
+          url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.12.tgz"
+          asc_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.12.tgz.asc"
+          sha512_url: "https://www.apache.org/dist/flink/flink-1.8.0/flink-1.8.0-bin-scala_2.12.tgz.sha512"
+      optional_components:
```

Comment:
optional components were intentionally decoupled from Flink releases. If flink-shaded-hadoop is moved to flink-shaded your approach no longer works.

Reply:
That's a good point! I think we will need both, then. Optional components that are tied to a release and optional components that are independent. Right now we only have dependent optional components.